src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -14,6 +14,8 @@ case class RawlsGroupName(value: String) extends UserAuthType
 case class RawlsGroupEmail(value: String) extends UserAuthType
 case class RawlsBillingProjectName(value: String) extends UserAuthType
 case class RawlsGroupMemberList(userEmails: Seq[String], subGroupEmails: Seq[String])
+case class RawlsUserInfo(user: RawlsUser, billingProjects: Seq[String])
```

Comment:
I prefer `Seq[RawlsBillingProjectName]` here.

`UserService.importUsers()` converts them to `RawlsBillingProjectName` anyway, so we might as well get the type-safety earlier.